src/views/Package/Package.tsx

```diff
@@ -46,13 +51,15 @@ export const Package: FunctionComponent = () => {
         )}
       </Box>
       {/* Readme and Api Reference Area */}
-      {assemblyResponse.data && !assemblyResponse.loading && (
-        <PackageDocs
-          assembly={assemblyResponse.data}
-          language={language}
-          submodule={q.get(QUERY_PARAMS.SUBMODULE) ?? ""}
-        />
-      )}
+      {markdownResponse.data &&
```

Comment:
Another non-blocker: I wonder if it would make sense to add a property to `useRequest`'s return value such as `loaded` so we can simplify these conditions